cypress/integration/specialFeatures/mediaPlayerPreRollAds/index.js

```diff
@@ -0,0 +1,51 @@
+import visitPage from '../../../support/helpers/visitPage';
+import getAppEnv from '../../../support/helpers/getAppEnv';
+import appToggles from '../../../support/helpers/useAppToggles';
+
+const pageType = 'mediaAssetPage';
+
+const embedUrlConfig = {
+  test: [
```

Comment:
Do we need to add a configuration for local environment (which points to polling.test...?)